doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
...no, I mean where does it show up in the actual documentation once built. That's what would give you an indication of how to link it properly from here so it renders as a link.